test/test_bort.F90

```diff
@@ -25,9 +25,13 @@ program test_bort
   real*8 real_1d(1)
   real*8 real_2d(1,1)
   integer idn30, idn30_val
-  integer :: num_args, len, status
+  integer :: num_args, len, stat, ios, u
   character(len=32) :: sub_name, test_case
   character*5 adn30
+  integer ibay(1), ibit, subset, jdate
+  integer mtyp, msbt, inod
+  character*28 unit
+  integer iscl, iref, nseq, nmsub
   
```

Comment:
I don't see where you have:

```
#ifdef KIND_8
  call setim8b(.true.)
#endif
```
anywhere in your test_bort.F90 code

So unless I'm missing something, this means that when you're doing `test_bort_8` calls from within run_test_bort.sh, then the integer*8 variables you're passing in to all of the various library routines aren't being treated as such inside of those library routines.  And which in turn means that many if not most of those `test_bort_8` calls are probably failing for reasons other than what you're really trying to test.

Reply:
OK, thanks for catching this! I fixed and at least one open issue went away (#401). So great!

Let's get this PR merged and we will press on with more testing!